### DISCUSSION TITLE
Live log window not getting feed from SatPC32

### DISCUSSION THREAD

**Discussion by geoham911** [magicbug/Cloudlog#2310] [General]
HI

Having a strange problem. Not been on Sats for about 4 months.  Fired everything back up and I have an issue.
My setup. Cloudlog running on a patched up to date Ubuntu box. Ver 20.
Using SatPC32 on Windows 11 computer. Tried both Cloudlog Aurora or SatPC32 Cloudlog Interface to get data into Cloudlog.

Everything looks ok and I can go into the hardware interfaces page and see the link and see it updates every few seconds with time/freq etc.

![image](https://github.com/magicbug/Cloudlog/assets/24734884/c97d9d41-0690-4d4e-81e5-93cd57ea62fa)

When I go to live log entry the settings for Sat, band, mode and freq etc used to sync up to the satellite selected but now they just sit at 160m AM

Am I doing something wrong.  Is there a simple way to tell which version I am on. Upgraded months ago to 2.4 when it was released and it was working ok when I was last operational.

Could a Linux upgrade have broken something.  Thinking about a new install on a new box to test but thought I would ask before doing that.

Regards

George
GI4SJQ

**Comment by magicbug** [comment]
In the General tab see radio and make sure satpc32 radio is selected

**Reply by geoham911** [reply]
HI Peter. I am such a dunce.  I know it was something simple.  Its 4 months since I was on sats and I dont know how it changes.  
Found it and set it and all working fine now.  
Thank you for coming back so quickly.

George

![image](https://github.com/magicbug/Cloudlog/assets/24734884/b37ca2e0-97af-4b68-9188-32cce3ecf157)

**Reply by magicbug** [reply]
Pleased that sorted it 👍🏻
